packages/angular/projects/ui-angular/yarn.lock

```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+tslib@^2.0.0:
```

Comment:
Should this yarn.lock file exist? I think there should be only one at the monorepo root.

Reply:
Oops, I'll delete this.